packages/jellyfish/src/jellyfish.ts

```diff
@@ -0,0 +1,14 @@
+import {JellyfishJsonRpc} from '@defichain/jellyfish-jsonrpc'
+
+export const Jellyfish = {
+  /**
+   * Initialize a DeFiChain client and use jsonrpc for interfacing
+   *
+   * @param url
+   */
+  jsonrpc(url: string): JellyfishJsonRpc {
+    return new JellyfishJsonRpc(url)
```

Comment:
I would call it `new Jellyfish()` if it makes sense so it's clean when it's used externally.

```js
new Jellyfish(url, {
    useNodeWallet: false, // default to false
}
```

Reply:
Definitely will pay more attention to this, I rewrote that part a few times.